hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -505,3 +505,36 @@ tests:
       - eval_time: 1m
         alertname: VMCannotBeEvicted
         exp_alerts: []
+
```

Comment:
I was expecting to find tests for `kubevirt_vmi_memory_domain_total_bytes` being renamed from `kubevirt_vmi_memory_used_total_bytes` here. Maybe you can add them?

Reply:
is there a way to implement such a test?

Reply:
I am confused. I was thinking that this yaml is a description of things to test, thus I'd have expected that you could just add another section for `kubevirt_vmi_memory_domain_total_bytes` like you did below this comment for the metric `kubevirt_vmi_memory_used_bytes`? If that is not the case, please elaborate.

Reply:
This PR does two things:

1. Rename "kubevirt_vmi_memory_used_total_bytes" metric as "kubevirt_vmi_memory_domain_total_bytes".
2. Introduce a new recording rule "kubevirt_vmi_memory_used_bytes" as "kubevirt_vmi_memory_available_bytes- kubevirt_vmi_memory_usable_bytes"

The details are available in the bug mentioned above.

I added some tests for the second part. That tests are useful for recording rules and alerts by using provided metrics in the test description.

I think there is no way to add a test for renaming. We can add tests for the metric by checking real value in environments with Prometheus but I thought it is out of scope of this fix. I added a new lane for metrics&alerts and there are some other+open tasks for completing the tests for all metrics&alerts.

Reply:
OK, understood! Thanks!